packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-add-step.ts

```diff
@@ -165,7 +165,7 @@ export class EmrAddStep extends sfn.TaskStateBase {
           'elasticmapreduce:DescribeStep',
           'elasticmapreduce:CancelSteps',
         ],
-        resources: [`arn:aws:elasticmapreduce:${Aws.REGION}:${Aws.ACCOUNT_ID}:cluster/*`],
+        resources: [`arn:${Aws.PARTITION}:elasticmapreduce:${Aws.REGION}:${Aws.ACCOUNT_ID}:cluster/*`],
```

Comment:
Use `Stack.formatArn()` API instead. This takes care of setting region, account and partition correctly.

Reply:
good call, will do